Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'pry-rails', '~> 0.3.4'
 gem 'rails_12factor', '~> 0.0.3', group: :production
 gem 'rack-cors', '~> 0.4.0', require: 'rack/cors'
 gem 'simple_token_authentication', '~> 1.6.0'
-
+gem 'paranoia'
```

Comment:
@MaicolBen I don't understand your comment. I mean Why `paranoia` is necessary?
(I know the answer but it's just for adding some context for other people)